src/utils/helpers.js

```diff
@@ -243,12 +243,30 @@ export const getAccessibleOnClickHandlers = onClick => {
     : undefined;
 };
 
+const rtlPlacement = (placement, rtl) => {
+  if (rtl || document.children[0].dir === 'rtl') {
```

Comment:
What about `document.documentElement` here instead of `document.children[0]`?

Reply:
Oh yeah, that does seem cleaner, thanks.